include/swift/Sema/ConstraintSystem.h

```diff
@@ -5394,6 +5394,7 @@ class OpenPackElementType {
 
     auto *elementType = cs.createTypeVariable(locator,
                                               TVO_CanBindToHole |
+                                              TVO_CanBindToNoEscape |
```

Comment:
One more for "extra" no-escape? :)

Reply:
Oops. Clearly I should drink my coffee before attempting to write any code 🙂